src/KeyboardLayout_pt_BR.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * Portuguese keyboard layout.
+ */
+
+#include "KeyboardLayout.h"
+
+extern const uint8_t KeyboardLayout_pt_BR[128] PROGMEM =
+{
+	0x00,          // NUL
+	0x00,          // SOH
+	0x00,          // STX
+	0x00,          // ETX
+	0x00,          // EOT
+	0x00,          // ENQ
+	0x00,          // ACK
+	0x00,          // BEL
+
+	0x2a,          // BS  Backspace
+	0x2b,          // TAB Tab
+	0x28,          // LF  Enter
+	0x00,          // VT
+	0x00,          // FF
+	0x00,          // CR
+	0x00,          // SO
+	0x00,          // SI
+	0x00,          // DEL
+	0x00,          // DC1
+	0x00,          // DC2
+	0x00,          // DC3
+	0x00,          // DC4
+	0x00,          // NAK
+	0x00,          // SYN
+	0x00,          // ETB
+	0x00,          // CAN
+	0x00,          // EM
+	0x00,          // SUB
+	0x29,          // ESC
```

Comment:
Again, for consistency with the other layouts, this should be `0x00`: the user is supposed to use `KEY_ESC` for the escape key.

That being said, using `"\e"` for the escape key is not a silly idea: this escape sequence is not standard C++, but is widely supported. If this is to be supported, however, it should be supported by all keyboard layouts, and that would be another change set.